docs/sdks/android/methods.md

```diff
@@ -325,3 +344,33 @@ Add the ChatView inside your Layout XML:
         android:layout_width="match_parent"
         />
 ```
+## Stories
+
+Add xml to your layout
```

Comment:
Please add 1-2 sentences of description what the stories are doing and also add 1-2 screenshots.

Images should be hosted here in this repo, not anywhere else.